7.1/Dockerfile

```diff
@@ -0,0 +1,110 @@
+# syntax = docker/dockerfile:experimental
+ARG SPRYKER_PHP_VERSION=7.1.27
+
+FROM php:${SPRYKER_PHP_VERSION}-fpm-alpine3.9
```

Comment:
You don't need three seperate Dockerfiles for this scenario. The only difference in dockerfiles is the `SPRYKER_PHP_VERSION` value. So you can de-duplicate the code, have only one `Dockerfile` and specify version on buildtime, like `docker build --build-arg (...)`

Reply:
Since we want use auto build option of Dockerhub and it can't do it with --build-arg, we've keep such version with additional ARG just to simplify further updates of each Dockerfiles (when PHP will roll out updated image). So we still need three Dockerfiles (discussed with Mike).